src/frontend/src/flows/manage/recoveryMethodsSection.ts

```diff
@@ -13,6 +14,8 @@ import { html, TemplateResult } from "lit-html";
 import { settingsDropdown } from "./settingsDropdown";
 import { Devices, RecoveryKey, RecoveryPhrase } from "./types";
 
+import copyJson from "./recoveryMethodsSection.json";
```

Comment:
Thanks! Not all modules used I18n, though those that do only use I18n whereas others inline the copy. Let's try and stay consistent within a module by either using I18n for the whole module or not using it at all

Reply:
I was not sure about it, for sure feel free to refactor when you are back. 

I assumed that I copied the exact same pattern as:

```
deviceSettings.json
deviceSettings.ts
```

Given the fact that all the files lives within the same packages without any particular naming distinction:

```
❯ pwd
/Users/daviddalbusco/projects/dfinity/internet-identity/src/frontend/src/flows/manage
❯ ls -ltr
total 128
-rw-r--r--@ 1 daviddalbusco  staff   2513 25 avr 18:27 migration.test.ts
-rw-r--r--@ 1 daviddalbusco  staff    744  7 mai 21:31 types.ts
-rw-r--r--@ 1 daviddalbusco  staff   1883 30 mai 08:51 settingsDropdown.ts
-rw-r--r--@ 1 daviddalbusco  staff   4998 26 jui 22:04 authenticatorsSection.ts
-rw-r--r--@ 1 daviddalbusco  staff   2785 26 jui 22:04 deviceSettings.json
-rw-r--r--@ 1 daviddalbusco  staff  11158 26 jui 22:04 deviceSettings.ts
-rw-r--r--@ 1 daviddalbusco  staff  15537 26 jui 22:04 index.ts
-rw-r--r--@ 1 daviddalbusco  staff   5031  6 jul 08:45 recoveryMethodsSection.ts
-rw-r--r--@ 1 daviddalbusco  staff    144  6 jul 08:45 recoveryMethodsSection.json
~/projects/dfinity/internet-identity/src/frontend/src/flows/manage fix/typo-you-enabled-a-recovery  
```

It's maybe just a naming issue to make more clear what's clear for you.

Anyway, like I said, I was not sure and just wanted to fix the bug. Go for a refactoring.